### PR TITLE
Display individual restaurant

### PR DESCRIPTION
**Pull request by JesperGreen**
PT Story: https://www.pivotaltracker.com/story/show/132161937

Changes proposed in this pull request:
- Display restaurant name, description and menus on restaurant show page.

What have I learned working on this feature:
- Using modal to call fake data and display it.
- Classes (inside tags) can have multiple names......

Screenshots: 
screen shot 2016-10-13 at 10 15 15 am

Ready for review!

### CODE REVIEW

File: www/js/services.js [hunk 1]
```diff
@@ -1,72 +1,78 @@
...
+            all: function () {
+                return restaurants;
+            }
+        };
     })
 
-.factory('restaurantsFactory', function($resource, API_URL) {
-  return $resource(API_URL + '/restaurants', {})
-});
+    .factory('restaurantsFactory', function ($resource, API_URL) {
```
Comment (diraulo) [inline www/js/services.js]: Is it really necessary to inject `API_URL` here? It's already defined as a global constant to the application so should be accessible from anywhere.

File: www/js/services.js [hunk 2]
```diff
@@ -1,72 +1,78 @@
 angular.module('slowfood.services', [])
 
-.factory('FakeRestaurants', function() {
+    .factory('FakeRestaurants', function () {
```
Comment (diraulo) [inline www/js/services.js]: Now that we're reading from the API, is this `FakeRestaurants` still needed?

**Review by diraulo — CHANGES_REQUESTED** [review]

File: www/js/services.js (hunk 1 above)
Comment (JesperGreen) [reply www/js/services.js]: Removing it breaks the entire app, I think we'll keep it for now?

File: www/js/services.js (hunk 2 above)
Comment (JesperGreen) [reply www/js/services.js]: It is not! Removed the fake data.

**Review by diraulo — APPROVED** [review]

**Pull request merged**